mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -518,6 +503,36 @@ def deactivate(self):
         self.bus.emit(msg.forward(f"intent.service.skills.deactivate",
                                   data={"skill_id": self.skill_id}))
 
+    def handle_converse_ack(self, message):
+        self.bus.emit(message.reply(
+            "mycroft.skill.converse.pong",
+            data={"skill_id": self.skill_id,
+                  "can_handle": self.converse_is_implemented},
+            context={"skill_id": self.skill_id}))
+
+    def handle_converse_request(self, message):
```

Comment:
Same as above, I think this should only be called internally by the event handler?

Reply:
i suppose it can be made private, but dont feel too strongly about either way. can't think of any circumstance where any skill would want to override this

Reply:
Mostly thinking that someone might not understand the intent here and shortcut overriding this method instead of `converse`